CRM/Pledge/BAO/PledgeBlock.php

```diff
@@ -61,8 +61,8 @@ public static function &create(&$params) {
    * Add or update pledgeBlock.
    *
    * @param array $params
-   *
-   * @return object
+   * @deprecated
+   * @return CRM_Pledge_DAO_PledgeBlock
    */
   public static function add($params) {
```

Comment:
OK - this has some non-api-worthy horror show but nothing the api needs

![image](https://user-images.githubusercontent.com/336308/180616314-c7249094-aab5-4d40-9fb1-7307c8adc060.png)